src/pool/srv_iv.c

```diff
@@ -1598,8 +1599,8 @@ ds_pool_iv_svc_fetch(struct ds_pool *pool, d_rank_list_t **svc_p)
 			D_GOTO(failed, rc);
 	} else {
 		/* create a ULT and schedule it on xstream-0 */
-		rc = dss_ult_execute(cont_pool_svc_ult, &ia, NULL, NULL,
-				     DSS_XS_SYS, DSS_ULT_DEEP_STACK, 0);
+		rc = dss_ult_execute(cont_pool_svc_ult, &ia, NULL, NULL, DSS_XS_SYS, 0,
+				     DSS_DEEP_STACK_SZ);
```

Comment:
I think this is wrong. DSS_DEEP_STACK_SZ=2 , it seems incorrect to pass the literal value 2 into this call stack, which will eventually invoke ABT_thread_attr_set_stacksize() as the stacksize argument.

Here is the one other example in the engine code that specifies a specific, nonzero stack_size argument, and it uses a number of bytes argument https://github.com/daos-stack/daos/blob/6dae36aad9d5d66ac39ce1a7aca9210d4d911808/src/object/srv_obj_migrate.c#L3264

Reply:
Actually, I stand corrected. I did not look closely enough to see that the code change removes the use of DSS_ULT_DEEP_STACK=2 and replaces it with DSS_DEEP_STACK_SZ=65536. Sorry about that.

Reply:
I think ds_dtx_resync() has the same issue, could you also add the change to this PR?

Reply:
Up to now, `ds_dtx_resync()` does not need deep stack size. But for `cont_pool_svc_ult()`, the original author explicitly pointed out that it required deep stack, but used wrong parameter.